product-service/src/handlers/getProductById.ts

```diff
@@ -0,0 +1,13 @@
+import 'source-map-support/register';
+import { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { getOneProductById } from "../services/products";
+import { response } from './utils';
+
+export const getProductById: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent) => {
+    try {
+        const product = await getOneProductById(event.pathParameters.productId);
+        return response(200, JSON.stringify(product));
+    } catch (e) {
+        return response(400, JSON.stringify(e));
```

Comment:
The same about response. Also would be nice to handle cases when there is no product with requestd id. And return 404 with a proper message.
At the moment it is looks like this: 
![image](https://user-images.githubusercontent.com/47124369/125613808-c830540b-5af5-401d-8b8a-083267ed74ad.png)
So it is hard to understand what went wrong